onwuvic/MyDiary
# #159501323 Add swagger documentation of the API

#### What does this PR do?
Swagger API Documentation for MyDiary API

#### Description of Task to be completed?
- Add swagger UI to the root folder
- Create a swagger.yaml file containing the documentation
- Create a route for the documentation

#### How should this be manually tested?
Visit https://tranquil-harbor-77266.herokuapp.com/api/v1/documentation to see the swagger doc

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
- Story type: chores
- Story title: Add Swagger Documentation
- Story ID: #159501323

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
